tests/runtime/paths/wasm.rs

```diff
@@ -0,0 +1,12 @@
+wit_bindgen::generate!({
```

Comment:
Adding a runtime test for this is pretty heavyweight and not much runtime functionality is tested here, so could this be added to `crates/rust/tests/*` instead?